File: LeetCode/27.Remove_Element/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    vector<int> v1 {3,2,2,3};
    int val1 = 3;
    vector<int> v2 {0,1,2,2,3,0,4,2};
    int val2 = 2;
    if (s.removeElement(v1, val1) == 2) { cout << "Pass" << endl;}
    else { cout << "Failed - v1" << endl;}
    if (s.removeElement(v2, val2) == 5) { cout << "Pass" << endl;}
    else { cout << "Failed - v2" << endl;}
    return 0;
}

File: LeetCode/27.Remove_Element/main.py
#!/usr/bin/python

from solution import Solution

# Unit Test

s = Solution()
if (s.removeElement([3,2,2,3], 3) == 2):
    print("Pass")
    s.reset_i(0)
else:
    print("Failed nums1")
if (s.removeElement([0,1,2,2,3,0,4,2], 2) == 5):
    print("Pass")
    s.reset_i(0)
else:
    print("Failed nums2")